src/Raven.Server/Web/Authentication/AdminCertificatesHandler.cs

```diff
@@ -226,13 +226,12 @@ public async Task Put()
 
                 if (LoggingSource.AuditLog.IsInfoEnabled)
                 {
-                    var clientCertificate = GetCurrentCertificate();
-                    var auditLog = LoggingSource.AuditLog.GetLogger("Certificates","Audit");
                     var permissions = certificate?.Permissions != null
                         ? Environment.NewLine + string.Join(Environment.NewLine, certificate.Permissions.Select(kvp => kvp.Key + ": " + kvp.Value.ToString()))
                         : string.Empty;
-                    auditLog.Info($"Add new certificate '{certificate?.Thumbprint}'. Security Clearance: {certificate?.SecurityClearance}. Permissions:{permissions}." +
-                                  $"{Environment.NewLine}IP: '{HttpContext.Connection.RemoteIpAddress}'. Certificate: {clientCertificate?.Subject} ({clientCertificate?.Thumbprint})");
+
+                    LogAuditFor("Certificates",
+                        $"Add new certificate '{certificate?.Thumbprint}'. Security Clearance: {certificate?.SecurityClearance}. Permissions:{permissions}.");
```

Comment:
We're missing cert subject - please format it `'certname' [thumbprint]`. In general when holding an instance of X509 certificate we should format it the same way presenting subject and thumbprint. Maybe need a helper method for that?

Reply:
done